examples/tutorial/c/tut_resources.c

```diff
@@ -37,14 +37,10 @@ int main()
   /* _sphinx_tag_tut_resource_types_start */
   allocate_and_deallocate("HOST");
 
-#if defined(UMPIRE_ENABLE_CUDA)
+#if defined(UMPIRE_ENABLE_CUDA) || defined(UMPIRE_ENABLE_HIP)
```

Comment:
Same comment as above - this is better to guard with the UMPIRE_ENABLE_DEVICE,UM,etc.